.gitignore

```diff
@@ -1 +1,26 @@
 tags
+obj/
+lib/
+alire/
+config/
+obj/
+lib/
+alire/
+config/
+obj/
+lib/
+alire/
+config/
+obj/
+lib/
+alire/
+config/
+obj/
+lib/
+alire/
+config/
+obj/
+lib/
+alire/
+config/
+bin/
```

Comment:
Some duplicates I think

Reply:
Ah, good catch. This was auto generated by alire but I had to run it multiple times

Reply:
Fixed in 6530dc1